src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -38,6 +38,7 @@ public class Wallet implements Serializable {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Setter
```

Comment:
What are you using this annotation for? Please, format your code.

Reply:
This is a mistake. Deleted.